internal/functions/efgs/api/structs.go

```diff
@@ -122,15 +122,20 @@ func (wrappedKey *DiagnosisKeyWrapper) ToData() *DiagnosisKey {
 		rt = 0 // This is an override for putting 0x7fffffff there
 	}
 
+	byteKeyData, err := base64.StdEncoding.DecodeString(wrappedKey.KeyData)
+	if err != nil {
+		return nil
```

Comment:
I believe we can assume that this is a VERY invalid state, something that should NEVER happen, so we can use `panic` here instead. I vote for:
1. Logging the situation (so it's correctly described in functions logs)
2. Panicking the whole processing